### ISSUE TITLE
Full Tearsheet is broken due to pandas update

### ISSUE THREAD

**Issue by tibkiss** [deltaray-io/mesosim-tracker#52]
Pandas just released 2.0.0, which breaks QuantStats:
QuantStats defines pandas dependency as `pandas>=0.24.0` 
which brings in the new 2.0.0 version of Pandas.

Our html report fails with:
`Error creating html report: pivot() takes 1 positional argument but 4 were given`

**Comment by tibkiss** [comment]
Last known working dependency list from 2023-03-31:
```
appdirs==1.4.4
beautifulsoup4==4.12.0
certifi==2022.12.7
cffi==1.15.1
charset-normalizer==3.1.0
contourpy==1.0.7
cryptography==40.0.1
cycler==0.11.0
fonttools==4.39.3
frozendict==2.3.6
html5lib==1.1
idna==3.4
importlib-resources==5.12.0
kiwisolver==1.4.4
lxml==4.9.2
matplotlib==3.7.1
multitasking==0.0.11
numpy==1.24.2
packaging==23.0
pandas==1.5.3
Pillow==9.4.0
pycparser==2.21
pyparsing==3.0.9
python-dateutil==2.8.2
pytz==2023.3
QuantStats==0.0.59
requests==2.28.2
scipy==1.10.1
seaborn==0.12.2
six==1.16.0
soupsieve==2.4
tabulate==0.9.0
urllib3==1.26.15
webencodings==0.5.1
yfinance==0.2.14
zipp==3.15.0
```

**Comment by tibkiss** [comment]
Fixed in version v2.6.4: The internal fix is to freeze all packages. 

Once https://github.com/ranaroussi/quantstats/pull/260 is merged or https://github.com/ranaroussi/quantstats/issues/259 resolved we can switch to the upstream fix.
